econml/bootstrap.py

```diff
@@ -212,21 +220,31 @@ def get_dist(est, arr):
                         return 2 * est - arr
                     else:
                         raise ValueError("Invalid kind, must be either 'percentile' or 'pivot'")
```

Comment:
Maybe here too we should add a test that the correct attribute error is raised.

Reply:
Refactored so that it's impossible to hit this line